pyseir/inference/fit_results.py

```diff
@@ -41,7 +68,9 @@ def load_inference_result(fips):
         Dictionary of fit result information.
     """
     output_file = get_run_artifact_path(fips, RunArtifact.MLE_FIT_RESULT)
-    df = pd.read_json(output_file, dtype={'fips': 'str'})
+    json_file = open(output_file, 'r')
+    d = simplejson.load(json_file, object_hook=lambda x: convert_fips_to_str(x))
```

Comment:
Why the switch to this?

Reply:
lots of i/o, some performance benefit

Reply:
Does this truncate leading zeros if it is being read in as an int?  These are really tiny files so I would argue the complexity of this is not worth it.

Reply:
ok, ill revert this